install/kubernetes/helm/istio/charts/certmanager/templates/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
 {{- end }}
       containers:
       - name: certmanager
-        image: "{{ .Values.hub }}/cert-manager-controller:{{ .Values.tag }}"
+        image: "{{ .Values.hub }}/{{ .Values.image }}:{{ .Values.tag }}"
```

Comment:
this looks like a third party dependency that was missed when we cleaned up the customizable values.  I know this isn't of much help for your use case, but would accept a PR that addresses this specific third party dependency.